2021/23xxx/CVE-2021-23872.json

```diff
@@ -68,6 +68,12 @@
                         "value": "CWE-269: Improper Privilege Management"
```

Comment:
An exception of type JSONDecodeError occurred. Arguments:

Reply:
Updated in second pull on same branch